tests/acceptance/topics-test.js

```diff
@@ -0,0 +1,86 @@
+import { test } from 'qunit';
+import moduleForAcceptance from 'atlantaember/tests/helpers/module-for-acceptance';
+
+const doesInclude = function (string, substrings) {
```

Comment:
This is neat! Did you write this? There should be a library that includes nifty things like this.